array-methods/map.js

```diff
@@ -0,0 +1,19 @@
+const prices = [42.42, 10, 28.2234, 3.2, 5, 12];
+
+const arr = prices.map(map, prices);
+console.log('sales', arr);
+function map(prices) {
+  const P = {
+    price: prices,
+    salePrice: prices / 2,
+  };
+  return P;
```

Comment:
Or you could just `return { .... };` and avoid the intermediate local variable. Or you could just use a shorthand return from the arrow function. (Remember from the arrow functions quiz how to return an object literal?)